<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="plotScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>774</width>
         <height>447</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave_Selections_As_2">
     <property name="title">
      <string>Save Selections As</string>
     </property>
     <addaction name="actionSave_List"/>
     <addaction name="actionSave_Stream"/>
     <addaction name="actionSave_Dat"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuSave_Selections_As_2"/>
    <addaction name="actionSave_Plot"/>
    <addaction name="actionSave_Filters"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <widget class="QMenu" name="menuHistogram_Bar">
     <property name="title">
      <string>Histogram / Bar</string>
     </property>
     <addaction name="actionReset"/>
    </widget>
    <addaction name="menuHistogram_Bar"/>
    <addaction name="actionScatter"/>
    <addaction name="action2D_Histogram"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowFilters"/>
    <addaction name="actionShowDatasetPanel"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlot"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_List">
   <property name="text">
    <string>CrystFEL List</string>
   </property>
  </action>
  <action name="actionSave_Stream">
   <property name="text">
    <string>Stream File</string>
   </property>
  </action>
  <action name="actionSave_Dat">
   <property name="text">
    <string>Dat File</string>
   </property>
  </action>
  <action name="actionSave_Plot">
   <property name="text">
    <string>Save Plot</string>
   </property>
  </action>
  <action name="actionScatter">
   <property name="text">
    <string>Scatter</string>
   </property>
  </action>
  <action name="action2D_Histogram">
   <property name="text">
    <string>2D Histogram</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionShowFilters">
   <property name="text">
    <string>Filters</string>
   </property>
  </action>
  <action name="actionSave_Filters">
   <property name="text">
    <string>Save Filters</string>
   </property>
  </action>
  <action name="actionShowDatasetPanel">
   <property name="text">
    <string>Dataset Panel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
